### PR TITLE
feat(index)

### PR DESCRIPTION
**Pull request by iamstarcode**
- Consume API
- Implement skeleton
- Rearrange to include dist folder deployment

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+      ? ' '
+      : `<li class="font-medium">Synonyms: “${flatArray(
+          definition.synonyms,
+        )}”</li>`
+  antonymsHtml +=
+    definition.antonyms == undefined
+      ? ' '
+      : `<li class="font-medium">Synonyms: “${flatArray(
+          definition.antonyms,
+        )}”</li>`
```
Comment (TheoOkafor) [inline dist/index.js:121]: if...else statement is more readable than ternary. Consider using it here

File: dist/index.js [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+  definitionHtml += exampleHtml + synonymsHtml + antonymsHtml + '</ul></li>'
+  return definitionHtml
+}
+
+function flatArray(arr) {
+  let word = ''
+  arr.map((item) => (word = word + item + ', '))
+  let i = word.lastIndexOf(', ')
+  word = word.substring(0, i)
+  return word.trim()
```
Comment (TheoOkafor) [inline dist/index.js:132]: arr.reduce() can accomplish the same thing without the need for line 130 and 131

Comment (TheoOkafor) [reply dist/index.js:132]: Try it out

Comment (iamstarcode) [reply dist/index.js:132]: Absolutelty will do, reduce will be more clean there.

File: dist/index.js (hunk 1 above)
Comment (iamstarcode) [reply dist/index.js:121]: Will do.

**Pull request merged**